README.md

```diff
@@ -50,74 +47,70 @@ This task overrides the path to the global `datadog-ci.config.json` file.
    - mobile_application_version_file_path: 'path/to/application.apk'
-   - site: 'datadoghq.com'
+   - site: {{< region-param key="dd_site" code="true" >}}
    - version_name: 'example 1.0'
 ```
 
 ## Inputs
 
 | Name                               | Requirement | Description                                                                                                                             |
 | -----------------------------------| :---------: | --------------------------------------------------------------------------------------------------------------------------------------- |
-| `apiKey`                           | _required_  | Your Datadog API key. This key is created by your [Datadog organization][3003] and will be accessed as an environment variable.         |
-| `appKey`                           | _required_  | Your Datadog application key. This key is created by your [Datadog organization][3003] and will be accessed as an environment variable. |
-| `configPath`                       | _optional_  | The global JSON configuration is used when launching tests. See the [example configuration][3002] for more details.                     |
+| `apiKey`                           | _required_  | Your Datadog API key. This key is created by your [Datadog organization][8] and will be accessed as an environment variable.         |
+| `appKey`                           | _required_  | Your Datadog application key. This key is created by your [Datadog organization][8] and will be accessed as an environment variable. |
+| `configPath`                       | _optional_  | The global JSON configuration is used when launching tests. See the [example configuration][9] for more details.                     |
 | `latest`                           | _optional_  | Marks the application as `latest`. Any tests that run on the latest version will use this version on their next run.                    |
 | `mobileApplicationVersionId`       | _required_  | ID of the application you want to upload the new version to.                                                                            |
-| `mobileApplicationVersionFilePath` | _required_  | Override the application version for Synthetic mobile application tests.                                                                |
-| `site`                             | _optional_  | The Datadog site to send data to. If the `DD_SITE` environment variable is set, it takes precedence.                                    |
+| `mobileApplicationVersionFilePath` | _required_  | Override the application version for [Synthetic mobile application tests][15].                                                                |
+| `site`                             | _optional_  | The [Datadog site][14] to send data to. Your default site is {{< region-param key="dd_site" code="true" >}}. If the `DD_SITE` environment variable is set, it takes precedence.                                    |
```

Comment:
Could we avoid using `{{ }}` markup in publicly available markdown documents? It'll be displayed as is. Instead I believe we should use `<!-- -->` markup, like [on the datadog-ci synthetics readme](https://github.com/DataDog/datadog-ci/blob/bff12e212215133d0eb59ae1814d052a2ee886c2/src/commands/synthetics/README.md?plain=1#L68).
```suggestion
| `site`                             | _optional_  | The [Datadog site][14] to send data to. <!-- partial Your Datadog site is {{< region-param key="dd_site" code="true" >}}. partial -->. If the `DD_SITE` environment variable is set, it takes precedence.                                    |
```